Blazor_Foundation_6_Test_Shared/Documentation/Containers/ResponsiveAccordionTabs.razor

```diff
@@ -1,7 +1,105 @@
+        <TabHeader Id="example-tabs">
+            <TabHeaderTitle Active="true" Id="panel-1">Tab 1</TabHeaderTitle>
+            <TabHeaderTitle Id="panel-2">Tab 2</TabHeaderTitle>
+            <TabHeaderTitle Id="panel-3">Tab 3</TabHeaderTitle>
+            <TabHeaderTitle Id="panel-4">Tab 4</TabHeaderTitle>
+            <TabHeaderTitle Id="panel-5">Tab 5</TabHeaderTitle>
+            <TabHeaderTitle Id="panel-6">Tab 6</TabHeaderTitle>
+        </TabHeader>
+        <TabContentWrapper LinkId="example-tabs">
+            <TabContent Active="true" Id="panel-1">
+                <p>one</p>
+                <p>Check me out! I'm a super cool Tab panel with text content!</p>
+            </TabContent>
+            <TabContent Id="panel-2">
+                <p>two</p>
+                <img class="thumbnail" src="https://get.foundation/sites/docs/assets/img/generic/rectangle-7.jpg">
+            </TabContent>
+            <TabContent Id="panel-3">
+                <p>three</p>
+                <p>Check me out! I'm a super cool Tab panel with text content!</p>
+            </TabContent>
+            <TabContent Id="panel-4">
+                <p>four</p>
+                <img class="thumbnail" src="https://get.foundation/sites/docs/assets/img/generic/rectangle-2.jpg">
+            </TabContent>
+            <TabContent Id="panel-5">
+                <p>five</p>
+                <p>Check me out! I'm a super cool Tab panel with text content!</p>
+            </TabContent>
+            <TabContent Id="panel-6">
+                <p>six</p>
+                <img class="thumbnail" src="https://get.foundation/sites/docs/assets/img/generic/rectangle-8.jpg">
+            </TabContent>
+        </TabContentWrapper>
+
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.Callout TypeClass="success">
```

Comment:
Example should be provided data-responsive-accordion-tabs is not supported it seems.
See TabHeader's component you will see data-responsive-accordion-tabs and that should be set like the Classes.

Reply:
Also test that  'responsiveness' to make sure we get the proper behavior